skills/messaging-help.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * When the bot joins a channel, tell everyone what it can do
+ */
+const config = require('../config');
+
+const helpMessage = `I'm here to help you find out what PRs you currently have open and which of them need your attention. In order to get the most out of me, you can use the following slash commands:
```

Comment:
Multi-line string literals are the best way to format messages from what I've found so far.